src/main/java/core/basesyntax/service/FruitInventoryManipulator.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service;
+
+import core.basesyntax.InvalidTransactionException;
+import core.basesyntax.db.FruitInventory;
+
+public interface FruitInventoryManipulator {
+    static Integer getValue(String fruit) {
+        return FruitInventory.getInventory().get(fruit);
+    }
+
+    static void add(String fruit, int quantity) {
+        Integer initialValue = getValue(fruit);
+        int useValue = initialValue == null ? 0 : initialValue;
+        FruitInventory.getInventory().put(fruit, useValue + quantity);
+    }
+
+    static void subtract(String fruit, int quantity) {
+        int initialValue = FruitInventory.getInventory().get(fruit);
+        validateValue(initialValue);
+        if (initialValue < quantity) {
+            throw new InvalidTransactionException("Not enough fruit in stock");
+        }
+        FruitInventory.getInventory().put(fruit, initialValue - quantity);
+    }
+
+    static void validateValue(Integer value) {
```

Comment:
Please avoid using static methods. Need to rework and use FruitInventoryManipulator instance as variable in classes that using it.
You will struggle during testing otherwise